### PR TITLE
OCPBUGS#4065: Policies have violations during deployment

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.12

Issue: https://issues.redhat.com/browse/OCPBUGS-4065

Link to docs preview: https://57926--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-known-issues

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by imiller0 — COMMENTED** [review]
LGTM

**Review by skrthomas — COMMENTED** [review]
A few minor comments for you to consider :)

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -3159,6 +3159,8 @@ $ oc delete pods --all -n openshift-cloud-controller-manager
 +
 There is currently no workaround to resolve this issue if encountered. To avoid this issue, xref:../machine_management/control_plane_machine_management/cpmso-disabling.adoc[disable the control plane machine set] and do not replace control plane machines manually if your cluster uses the OVN-Kubernetes network provider. (link:https://issues.redhat.com/browse/OCPBUGS-5306[*OCPBUGS-5306*])
 
+* If a cluster that was deployed through ZTP has policies that do not become compliant and no `ClusterGroupUpdates` object is present, you need to restart the {cgu-operator} pods. Restarting {cgu-operator} creates the proper `ClusterGroupUpdates` object which enforces the policies and the policies become compliant. (link:https://issues.redhat.com/browse/OCPBUGS-4065[*OCPBUGS-4065*])
```
[the remark this answers is not in the document]
Comment (amolnar-rh) [reply release_notes/ocp-4-12-release-notes.adoc]: Thank you for the explanations!

I added almost everything, except the definite article before the Operator. `the {cgu-operator}` will become `the TALM` and we've been using TALM as abbreviation without a def article. I hope that makes sense!

**Review by jldohmann — APPROVED** [review]
LGTM

**Pull request merged**